Plutarch/Rational.hs

```diff
@@ -1,3 +1,4 @@
+{-# OPTIONS_GHC -Wno-incomplete-patterns #-}
```

Comment:
?

Reply:
I don't think we need the other pragma either.

Reply:
This is a _really_ bad thing to enable in general.

Reply:
Sorry about that, I think the haskell-language-server plugin I'm trying out added those. I'm not sure why. I've removed both pragmas.